### ISSUE TITLE
Implement Design Tokens and Specifications in Gitcoin Passport App

### ISSUE THREAD

**Issue by erichfi** [gitcoinco/passport#1054]
Implement the new design tokens and specifications in the Passport app to ensure consistency and improve user experience. If necessary based on technical assessment, we break down down the task into more manageable sub-tasks.

Tasks:

- [x] Review the design tokens and specifications document provided by the designer (available at NOTION LINK).
- [x] Break down the design tokens and specifications into categories (e.g., colors, typography, spacing, icons).
- [x] Assess the complexity of implementing each category and determine if it warrants a separate issue.
- [x] Create any necessary additional issues, linking them to this main issue for better tracking and organization.
- [ ] Implement the design tokens and specifications in the Gitcoin Passport app.
- [ ] Perform thorough testing to ensure consistency across different browsers and devices.
- [ ] Update the team on the progress and any challenges encountered during the implementation.

Acceptance Criteria:

- Design tokens and specifications are implemented according to the designer's guidelines.
- UI elements in the Gitcoin Passport app adhere to the design tokens and specifications.
- The app's user experience is consistent across different browsers and devices.
- The product development team is updated on progress and any potential roadblocks.

This depends on #1191

**Comment by Etovardesign** [comment]
Design Token Docs:

[Notion Doc.](https://www.notion.so/gitcoin/Passport-Holder-s-Experience-UI-Specifications-cd7ec22ecfb74c3e8794f17eb2b6bcd8?pvs=4)

[Figma Styleguide](https://www.figma.com/file/61l4GXuZTNIR8gfSybFqfo/Gitcoin-Passport---Lunar?node-id=66%3A2021&t=5L0Ns1qRY0RCBhUb-1)

[Figma Designs](https://www.figma.com/file/ShlrWEZ3SVjYi31q0ehDva/Passport-S17?node-id=2429%3A85054&t=BckjgzsiSJii9qLn-1)

**Comment by lucianHymer** [comment]
Fonts: https://github.com/gitcoinco/passport/issues/1272
Spacing: https://github.com/gitcoinco/passport/issues/1273
Switches, Checkboxes, Buttons: https://github.com/gitcoinco/passport/issues/1274
Macro Components: https://github.com/gitcoinco/passport/issues/1275

Once these are all complete, a final overview will be done and then this ticket closed out.
